tests/config.yaml

```diff
@@ -7,4 +7,5 @@ modules:
     dry: true
     dependencies: false
 repos:
+ - ""
  - git@github.com:StartupOS/small-test-repo.git
```

Comment:
Would length be 0 for this?

Reply:
Yup, the quotes force yaml to interpret it as an empty string. Without the quotes it's invalid yaml.